src/commands/checkCommands.ml

```diff
@@ -87,11 +87,15 @@ module CheckCommand = struct
     if Options.should_profile options
     then begin
       Flow_server_profile.init ();
-      let rec sample_processor_info () =
-        Flow_server_profile.processor_sample ();
-        Timer.set_timer ~interval:1.0 ~callback:sample_processor_info |> ignore
-      in
-      sample_processor_info ()
+      if Sys.win32 then
+        Flow_server_profile.processor_sample ()
```

Comment:
I was trying to work out whether it makes sense to call this once or not ;- but after 30 minutes searching github and google I cannot work out where this is implemented :(